crates/iota-graphql-rpc/src/types/transaction_block/mod.rs

```diff
@@ -196,6 +243,7 @@ impl TransactionBlock {
 
     /// The type of this transaction as well as the commands and/or parameters
     /// comprising the transaction of this kind.
+    #[graphql(complexity = 0)]
```

Comment:
e.g. if kind is genesis tx, then user can query events of this tx, and from there query txs, and so on